code/headers/gga.hpp

```diff
@@ -3,6 +3,17 @@
 #include <hwlib.hpp>
 
 namespace r2d2::location {
+    /**
+     * @brief Struct containing the degrees, minutes, tenthousends minute for
+     * latitude/longitude.
+     *
+     */
+    struct degrees {
```

Comment:
Is it really a good idea to call the structs degrees when it has a member of the same name?
It also isn't really descriptive.

Reply:
changed the name to `decimal_degrees`.